gpumon/gpumon_interface.ml

```diff
@@ -0,0 +1,28 @@
+(*
+ * Copyright (C) Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+let service_name = "gpu"
+let queue_name = Xcp_service.common_prefix ^ service_name
+let json_path = "/var/xapi/" ^ service_name ^ ".json"
+let xml_path = "/var/xapi/" ^ service_name
```

Comment:
`xml_path` is used by the `xml_url` function in `gpumon/gpumon_client.ml`, which is used in the `xml_http_rpc` call below that in the same file. `json_path`, as @robhoes pointed out above, isn't used, so I'm removing it.